source/scss/lark.scss

```diff
@@ -0,0 +1,16 @@
+/*
+ * Lark
+ *
+ */
+@import 'shared';
+
+%lark,
+.lark {
+  @extend %filter-base;
+  filter: contrast(0.9);
+
+  &:before {
+    background: #22253f;
```

Comment:
As per #97 -- we're going to use consistency via color names in RGB from now on. Please convert this hex code